<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>400</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTextEdit" name="textEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>MAKE QR</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>READ QR</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>SCAN QR</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>770</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMENU">
    <property name="title">
     <string>MENU</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionLOAD"/>
    <addaction name="separator"/>
    <addaction name="actionSAVE"/>
    <addaction name="separator"/>
    <addaction name="actionQUIT"/>
   </widget>
   <addaction name="menuMENU"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLOAD">
   <property name="text">
    <string>LOAD</string>
   </property>
  </action>
  <action name="actionSAVE">
   <property name="text">
    <string>SAVE</string>
   </property>
  </action>
  <action name="actionQUIT">
   <property name="text">
    <string>QUIT</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
